src/OpenLoco/src/MessageManager.cpp

```diff
@@ -217,7 +217,8 @@ namespace OpenLoco::MessageManager
                 auto* town = TownManager::get(static_cast<TownId>(message.itemSubjects[2] & 0xFF));
                 args.push(town->name);
 
-                const auto formatStr = cargoObj->flags & CargoObjectFlags::unk2 ? StringIds::message_first_string_delivery_arives_at : StringIds::message_first_string_arrive_at;
+                const auto isDelivering = cargoObj->hasFlags(CargoObjectFlags::unk2);
+                const auto formatStr = isDelievering ? StringIds::message_first_string_delivery_arives_at : StringIds::message_first_string_arrive_at;
```

Comment:
This needed changing as well:
```suggestion
                const auto formatStr = isDelivering ? StringIds::message_first_string_delivery_arives_at : StringIds::message_first_string_arrive_at;
```